docs/data/data-grid/editing/editing.md

```diff
@@ -345,7 +345,24 @@ function CustomEditComponent(props: GridRenderEditCellParams) {
     apiRef.current.setEditCellValue({ id, field, value: newValue });
   };
 
-  return <input type="text" value={value} onValueChange={handleValueChange}>;
+  const handleBlur = (event: React.FocusEvent<HTMLInputElement>) => {
+    // The call below ensures that the props will be validated at least once
+    // Here it's done during the blur but it can also be done when the component is mounted
+    // Use it in an effect as follow instead:
+    // React.useEffect(() => {
+    //   apiRef.current.ensurePreProcessEditCellPropsRanOnce({ id, field });
+    // }, [apiRef, id, field]);
+    apiRef.current.ensurePreProcessEditCellPropsRanOnce({ id, field });
+  };
+
+  return (
+    <input
+      type="text"
+      value={value}
+      onBlur={handleBlur}
```

Comment:
What if we pass `onBlur` callback to `renderEditCell`, so that users don't have to use `ensurePreProcessEditCellPropsRanOnce` directly?
I find the method name non-intuitive, but at the same time I don't have better idea how to name it.
But `onBlur` is easier to reason about.

```suggestion
  // `onBlur` ensures that the props will be validated at least once
  // Here it's done during the blur but it can also be done when the component is mounted
  // Use it in an effect as follow instead:
  // React.useEffect(() => {
  //   onBlur();
  // }, [onBlur]);

  return (
    <input
      type="text"
      value={value}
      onBlur={onBlur}
```

Reply:
+1 if that's a feasible solution.
"ensurePreProcessEditCellPropsRanOnce" feels a bit hacky.